tests/lib/modem_trace/modem_trace_rtt/src/modem_trace_test.c

```diff
@@ -127,7 +123,7 @@ void test_modem_trace_init_rtt_transport_medium(void)
 static void expect_rtt_write(const uint8_t *buffer, uint16_t size)
 {
 	__wrap_SEGGER_RTT_ASM_WriteSkipNoLock_ExpectAndReturn(trace_rtt_channel, buffer, size,
-							size);
```

Comment:
Unneeded changes here? (applies to the next 3 changes in this file too)

Reply:
Formatting was quite off, I used `clang-format` on this file.